### DISCUSSION TITLE
Respawn location: Towny

### DISCUSSION THREAD

**Discussion by 0xC1A551C** [Zrips/CMI#922] [Feature Requests]
Would be nice if you could add the ability for the player to respawn at their townspawn (have an option like spawn, homespawn, bedspawn, ect)

**Relevant plugins (Delete if this isn't needed):**  Towny

**Comment by Tsumitsuki** [comment]
bit of an old one but would love this, too.

adding the town spawn betwen bedLocation and worldSpawn would be awesome.
